services/client/src/utils/get-buckets-for/index.js

```diff
@@ -1,17 +1,14 @@
 import _ from "lodash";
 import moment from "moment";
 
-const sameTimePeriod = period => ({ time }) =>
-  moment().isSame(moment(time), period);
-
 const withinDays = days => ({ time }) =>
   moment(time).diff(moment(), "days") <= days;
 
 const buckets = [
-  { id: "today", check: sameTimePeriod("day"), message: "Today" },
+  { id: "today", check: withinDays(1), message: "Today" },
   {
     id: "this-week",
-    check: sameTimePeriod("week"),
+    check: withinDays(7),
```

Comment:
Make sense - the "same day" / "same week" was by design, but I guess it doesn't really make much sense when viewed at the weekend 🤦‍♂️

Reply:
Ah 💡! That makes sense now.  I had assumed `This Week` was `Upcoming events in the next week`.  I was a little threw off by events in the next 4 day's appearing under `Next 30 days` and `This week` being empty.